src/main/appengine/index.yaml

```diff
@@ -35,6 +35,14 @@ indexes:
     name: isPublishedEmailEnabled
   - direction: asc
     name: sentPublishedEmail
+  - direction: asc
+    name: resultsVisibleFromTime
```

Comment:
I don't see here a query that requires a combination of these fields. Any reason this is added?

Reply:
Sorry, this was needed in one of the iteration of the session search logic. Can remove now.